src/components/pageTitle/pageTitle.scss

```diff
@@ -0,0 +1,7 @@
+.okp4-page-title {
+  padding-left: 5vw;
+
+  @media screen and (max-width: 1300px) {
```

Comment:
Imo the title should be centered when filter moves above the list. Same for the list configuration.

It looks weird to have the config and title not centered when the list is.